doc/book/zend.barcode.objects.md

```diff
@@ -0,0 +1,408 @@
+|text              |String                           |NULL                 |The text to represent as a barcode                                                                         |
+
+ ..
+\_zend.barcode.barcode.common.options.barcodefont:
+
+### Particular case of static setBarcodeFont()
+
+You can set a common font for all your objects by using the static method
+`Zend\Barcode\Barcode::setBarcodeFont()`. This value can be always be overridden for individual
+objects by using the `setFont()` method.
+
+```php
+<?php
+use Zend\Barcode\Barcode;
+
+// In your bootstrap:
+Barcode::setBarcodeFont('my_font.ttf');
+
+// Later in your code:
+Barcode::render(
+    'code39',
+    'pdf',
+    array('text' => 'ZEND-FRAMEWORK')
+); // will use 'my_font.ttf'
+
+// or:
+Barcode::render(
+    'code39',
+    'image',
+    array(
+        'text' => 'ZEND-FRAMEWORK',
+        'font' => 3
+    )
+); // will use the 3rd GD internal font
+
+```
+
+Common Additional Getters
+-------------------------
+
+
+|Getter                             |Data Type|Description                                                                                                            |
+|-----------------------------------|---------|-----------------------------------------------------------------------------------------------------------------------|
+|getType()                          |String   |Return the name of the barcode class without the namespace (e.g. Zend\\Barcode\\Object\\Code39 returns simply "code39")|
+|getRawText()                       |String   |Return the original text provided to the object                                                                        |
+|getTextToDisplay()                 |String   |Return the text to display, including, if activated, the checksum value                                                |
+|getQuietZone()                     |Integer  |Return the size of the space needed before and after the barcode without any drawing                                   |
+|getInstructions()                  |Array    |Return drawing instructions as an array.                                                                               |
+|getHeight($recalculate = false)    |Integer  |Return the height of the barcode calculated after possible rotation                                                    |
+|getWidth($recalculate = false)     |Integer  |Return the width of the barcode calculated after possible rotation                                                     |
+|getOffsetTop($recalculate = false) |Integer  |Return the position of the top of the barcode calculated after possible rotation                                       |
+|getOffsetLeft($recalculate = false)|Integer  |Return the position of the left of the barcode calculated after possible rotation                                      |
+
+
+
+orphan  
```

Comment:
Oh my bad, let me give this another go later no idea why the headings didn't work but I looked at this so long I was missing obvious stuff.